addons/website/static/src/scss/website.scss

```diff
@@ -211,6 +211,10 @@ body:not(.o_fullscreen) {
     }
     .nav-item {
         transition: opacity 1000ms ease 0s;
+
+		img.rounded-circle {
+		    object-fit: cover;
+		}
```

Comment:
This rule this looks too generic, and I'm actually not sure to understand what it is trying to target?

Reply:
But if other persons have customization that add other rounded image in the navbar and they don't want `object-fit: cover` on them this will break their current style...

For simple understanding, can you give me the link on github to the template line of the code with the `<img>` tag that that you try to fix with this change?

Reply:
@seb-odoo Done bro, please review